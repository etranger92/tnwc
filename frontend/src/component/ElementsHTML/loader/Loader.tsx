/* eslint-disable @typescript-eslint/no-unused-vars */
import React from "react";
import "./loader.css";

type TProps = {
  color?: string;
};
// eslint-disable-next-line no-unused-vars
const Loader: React.FC<TProps> = (props) => {
  //const { clickEvent } = props;
  return (
    <div id="loader">
      <div className="lds-roller">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
      </div>
    </div>
  );
};
export default Loader;
