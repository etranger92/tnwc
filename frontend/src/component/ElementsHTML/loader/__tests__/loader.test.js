import * as React from 'react';
import Loader from "../Loader.tsx"
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe("We test the innerHTML of the Loader", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Loader/>);
  });
  test("Snapshot Loader", () => {
    //It works
 expect(toJson(wrapper)).toMatchSnapshot();

});
});
