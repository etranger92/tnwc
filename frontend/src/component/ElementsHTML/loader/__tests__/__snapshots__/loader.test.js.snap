// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`We test the innerHTML of the Loader Snapshot Loader 1`] = `
<div
  id="loader"
>
  <div
    className="lds-roller"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
